packages/ui/src/components/va-time-picker/VaTimePicker.demo.vue

```diff
@@ -13,29 +13,43 @@
     </VbCard>
 
     <VbCard title="view">
-      <VaTimePicker v-model="value" :view="'seconds'" />
-      <br />
-      <VaTimePicker v-model="value" :view="'hours'" />
+      <div class="d-flex">
+        <VaTimePicker v-model="value" :view="'seconds'" />
+        <va-divider vertical />
+        <VaTimePicker v-model="value" :view="'hours'" />
+      </div>
     </VbCard>
 
     <VbCard title="ampm">
-      <VaTimePicker v-model="value" :ampm="ampm" />
-      <br />
-      <VaTimePicker v-model="value" :ampm="ampm" />
+      <div class="d-flex">
+        <VaTimePicker v-model="value" :ampm="ampm" />
+        <va-divider vertical />
+        <VaTimePicker v-model="value" :ampm="ampm" />
+      </div>
     </VbCard>
 
     <VbCard title="value">
-      23:05:53
-      <br />
-      {{ value.getHours() }}:{{ value.getMinutes() }}:{{ value.getSeconds() }}
+      <div class="d-flex justify--space-around">
+        <span>23:05:53</span>
```

Comment:
what is it?

Reply:
If you look the data, you'll see hardcoded time.